src/data-access/script.sql

```diff
@@ -6,3 +6,13 @@ insert into users (login, password, age) values ('petr_petrov', 'petrov1', 30);
 insert into users (login, password, age) values ('ivan_ivanov', 'ivanov1', 25);
 
 SELECT * FROM users;
+
+
+create table groups (id UUID PRIMARY KEY DEFAULT uuid_generate_v4(), name text, permissions text);
```

Comment:
Would be nice if you used sequelize-cli and created migrations

Reply:
Sequelize-cli package has been added and used.